static/js/components/EmptyState.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from "react";
+import { Button, Col, Icon, Row } from "@canonical/react-components";
+
+interface Props {
+  iconName: string;
+  iconClass: string;
+  title: string;
+  message: string;
+  linkMessage: string;
+  linkURL: string;
+  buttonLabel: string;
+  buttonAction: () => void;
+}
+
+const EmptyState: FC<Props> = ({
+  iconName,
+  iconClass,
+  title,
+  message,
+  linkMessage,
+  linkURL,
+  buttonLabel,
+  buttonAction,
+}) => {
+  return (
+    <Row>
+      <Col size={3} />
```

Comment:
We shouldn't use empty cols, better use `col-start-large-3 col-start-medium-2` or something like that from https://vanillaframework.io/docs/patterns/grid